src/plugins/docusaurus-plugin-matamo/analytics.js

```diff
@@ -22,6 +23,11 @@ export default (function () {
         ? location.pathname + location.search + location.hash
```

Comment:
If we just use pathname and omit search & hash, wouldn't this fix the issue also?

For more info see also:

https://github.com/facebook/docusaurus/blob/a41a5c328c6ed4c106659d06957dd2c99c3dd082/packages/docusaurus-plugin-google-gtag/src/gtag.ts#L20-L33